proto/hummock.proto

```diff
@@ -39,9 +39,6 @@ message Level {
 
 message UncommittedEpoch {
   uint64 epoch = 1;
-  // We store SstableInfo here because key range information is needed to update
-  // CompactStatus.
-  // TODO: should we combine CompactStatus with HummockVersion?
   repeated SstableInfo tables = 2;
```

Comment:
Btw, we can also change this back to sst id (u64) since the key range information is no longer needed. I am also okay of keeping it as it is since I think we will eventually remove `UncommittedEpoch` in `HummockVersion` after we remove `add_tables` and `get_table_id` RPCs.

Reply:
These SstableInfo is also used to filter out SSTables in https://github.com/singularity-data/risingwave/pull/1558. So I'd like to keep it in this PR.